server/webdriver_tests/place_explorer_test.py

```diff
@@ -37,6 +40,43 @@ def test_page_serve(self):
             self.assertEqual(response.getcode(), 200)
         self.assertEqual("United States | Place Explorer | Data Commons", self.driver.title)
 
+    def test_place_search(self):
+        """Test the place search box can work correctly."""
+        self.driver.get(self.url_ + USA_URL)
+        # Using implicit wait here to wait for loading page.
+        self.driver.implicitly_wait(5)
+        search_border = self.driver.find_element_by_class_name("search")
```

Comment:
can remove this line and find the search_box direclty?

Reply:
Done

Reply:
I don't see a new commit yet. Can you upload the commit? Usually the flow is to update a new commit first and respond to previous comments.

Reply:
Oh, I sent a question to you in Google chat, you can have a look :)